petab/measurements.py

```diff
@@ -154,7 +154,7 @@ def get_unique_parameters(series):
 
 def split_parameter_replacement_list(
         list_string: Union[str, numbers.Number],
-        delim: str = ';') -> List[Union[str, numbers.Number]]:
+        delim: str = PARAMETER_SEPARATOR) -> List[Union[str, numbers.Number]]:
```

Comment:
does it ever make sense to pass anything but `PARAMETER_SEPARATOR` here?

Reply:
Probably not for now. *If* the separator ever changes, it can be useful, but shouldn't hurt keeping the optional argument.

Reply:
but why is it optional for this specific function (in contrast to all the other code, where it isn't optional)?

Reply:
Because adding it to the whole potential call stack will be a mess :see_no_evil:

Reply:
🥇